### ISSUE TITLE
Forms are submitted when pressing Enter

### ISSUE THREAD

**Issue by dgdavid** [yast/d-installer#226]
Forms are submitted when the user press <kbd>Enter</kbd>, which is the expected behavior for HTML forms. However, our forms in these Popup/Modal dialogs does not have an _action_ set, ending up reloading the page.

The fix should be rather easy, just preventing the default event when the form is submitted, triggering the expected action instead.

**Comment by dgdavid** [comment]
See https://html.spec.whatwg.org/#implicit-submission

**Comment by dgdavid** [comment]
Taking the [`RootPassword`](https://github.com/yast/d-installer/blob/master/web/src/RootPassword.jsx) component as an example, a way to make it work nicely would be adding it a submit button. Since the `Popup.Confirm` action is already there we could simply use the [button](https://html.spec.whatwg.org/multipage/form-elements.html#the-button-element) `form` and `type` attributes for fixing the implicit submission.

I.e., 

```diff
  return (
    <>
      {renderLink()}


      <Popup
        isOpen={isFormOpen}
        aria-label="Set new root password"
      >
-        <Form>
+        <Form id="root-password-form">
          <FormGroup fieldId="rootPassword" label="New root password">
            <TextInput
              id="rootPassword"
              type="password"
              aria-label="root password"
              value={rootPassword}
              onChange={setRootPassword}
            />
          </FormGroup>
        </Form>


        <Popup.Actions>
-          <Popup.Confirm onClick={accept} isDisabled={rootPassword === ""} />
+          <Popup.Confirm form="root-password-form" type="submit" onClick={accept} isDisabled={rootPassword === ""} />
          <Popup.Cancel onClick={close} />
          <Popup.AncillaryAction onClick={remove} isDisabled={!isRootPasswordSet} key="unset">
            Do not use a password
          </Popup.AncillaryAction>
        </Popup.Actions>
      </Popup>
    </>
  );
```

**Comment by imobachgs** [comment]
This issue will be addressed as part of #227.
